addons/hr_holidays/views/hr_leave_type_views.xml

```diff
@@ -8,8 +8,10 @@
             <search string="Search Time Off Type">
                 <field name="name" string="Time Off Types"/>
                 <field name="create_calendar_meeting"/>
+                <field name="country_id" string="Country"/>
```

Comment:
I don't think that line is needed

Reply:
Yes but I'm not sure adding this field really impacts anything :thinking: You can add the filter on any field using custom filter. Or maybe I'm missing something? I'm not entirely sure of what adding fields here is supposed to do in the first place